Components/Navigation/navigation.css

```diff
@@ -0,0 +1,287 @@
+@import url('/CSS/styleGuide.css');
+@import url('/CSS/sideBar.css');
+@import url('/CSS/home.css');
+@import url('/CSS/reset.css');
```

Comment:
Rather than importing specific css files, you can just drop all css files in a single main one. And import that main in component.